DEPLOYMENT.md

```diff
@@ -0,0 +1,41 @@
+## Deployments
+
+### Avalanche - Joe Strategy
+
+| Contract           | Token address                              | Strategy Address                           |
+| ------------------ | ------------------------------------------ | ------------------------------------------ |
+| USDC/AVAX strategy | 0xA389f9430876455C36478DeEa9769B7Ca4E3DDB1 | 0x663Ef4455A07243D9029bA0fC48297AE181aeb38 |
+| MIM/AVAX strategy  | 0x781655d802670bbA3c89aeBaaEa59D3182fD755D | 0x43838338F30795185Dabf1e52DaE6a3FEEdC953d |
+| AVAX/USDT strategy | 0xeD8CBD9F0cE3C6986b22002F03c6475CEb7a6256 | 0x87A5bF86D6C96775d926F43700c0fD99EE0c2E82 |
+
+### Polygon - Aave Strategies
+
+| Contract             | Token address                              | Strategy Address                           |
+| -------------------- | ------------------------------------------ | ------------------------------------------ |
+| WETH Aave strategy   | 0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619 | 0x760ef4F484EbF2668001B090291f84A3CDf2f3aa |
+| USDC Aave strategy   | 0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174 | 0xCFb49550ce8f39c29E73BA0baBc16609A63b31b1 |
+| WBTC Aave strategy   | 0x1BFD67037B42Cf73acF2047067bd4F2C47D9BfD6 | 0xF2CF8109d1c66112132180c0d0f925bDD7b4c246 |
+| WMATIC Aave strategy | 0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270 | 0x7166D2efffCA02c6A21A235732131660c3E61f9F |
+| USDT Aave strategy   | 0xc2132D05D31c914a87C6611C10748AEb04B58e8F | 0x59c7459281B4CF2eE96C1CA2A410Fdf03F51A369 |
+| DAI Aave strategy    | 0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063 | 0xB6665Ba83e054A91db1e6fC2252b4346a12C60d7 |
+| AAVE Aave strategy   | 0xD6DF932A45C0f255f85145f286eA0b292B21C90B | 0x5fB5d087A67d412350060c848b826B9Fb0FE92bA |
+
+| Strategy                | Harvester address                          |
+| ----------------------- | ------------------------------------------ |
+| Polygon Aave strategies | 0xdC56bD1F319DC1976FD179Ba2A6A76363316a374 |
+
+### BSC - PancakeSwap Strategies
+
+| Contract      | Token address                              | Strategy Address                           |
+| ------------- | ------------------------------------------ | ------------------------------------------ |
+| Cake strategy | 0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82 | 0x08B918dD18E087893bb9d711d9E0BBaA7a63Ef63 |
+
+## Avalanche - Popsicle Dynamic LP Strategies
+
+**USDC.e/WAVAX**
+
+| Contract                                    | Strategy Address                           |
+| ------------------------------------------- | ------------------------------------------ |
+| Popsicle_UsdceWavaxJLP_DynamicLPStrategy    | 0x0D2a5107435cbbBE21Db1ADB5F1E078E63e59449 |
```

Comment:
The strategy should be owned by Abra AVAX multisig

Reply:
It's using Limone with the Popsicle Gnosis MultiSig, is that ok?